modules/ossm-using-discoveryselectors-scope-service-mesh.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+// install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-discoveryselectors-scope-service-mesh_{context}"]
+= Using DiscoverySelectors to scope a Service Mesh
+:context: ossm-discoveryselectors-scope-service-mesh
+
+If you know which namespaces to include in the {SMProductShortName}, configure `discoverySelectors` during or after installation by adding the required selectors to the MeshConfig resource of {istio}. For example, configure {istio} to discover only namespaces labeled `istio-discovery=enabled`.
+
+.Prerequisites
+* The {SMProduct} operator is installed
+* An {istio} CNI resource is created
+
+.Procedure
+. Add a label to the namespace containing the {istio} control plane, for example, the `istio-system` system namespace.
++
+[source,terminal]
+----
+$ oc label namespace istio-system istio-discovery=enabled
+----
+
+. Modify the `{istio}` control plane resource to include a `discoverySelectors` section with the same label.
++
+[source,yaml]
+----
+kind: Istio
```

Comment:
It looks like there were some issues copying over this example code... See how the example config was updated [draft doc here](https://github.com/openshift-service-mesh/sail-operator/tree/main/docs/ossm/create-mesh) - procedure step 2. 

The `Istio` CRD could be quite large with a lot of config that's not relevant, so I wasn't sure how much we should show. Looking at other examples, it looks like the common thing is to show the resource headers  (apiVersion, kind, maybe metadata) and then only the relevant configuration that needs to be updated, but I'll look for your guidance here.

This part should definitely not be included though, as it's not relevant to the code being changed and could cause problems if the user overwrites their existing config with these values:
```
updateStrategy:
        type: InPlace
      version: v1.23.0
```

Reply:
@longmuir Agree with your points. I have updated the example config following the draft doc. For the last point, I have a suggestion. If the user needs to change the version then it can be added as a step (wherever you need it). For example:

. Change the version in the {istio} resource:
+
[source,yaml]
----
kind: Istio
spec:
  version: 1.23.2
  updateStrategy: 
    type: InPlace
 ----
 
If you think that's not needed, I can ignore the comment. Do let me know what you think! :)